### DISCUSSION TITLE
Python Development Environments and Functions for Machine Learning and Data Science

### DISCUSSION THREAD

**Discussion by vsfeedback** [Microsoft/PTVS#7335] [Enchancements]
_This issue has been moved from [a ticket on Developer Community](https://developercommunity.visualstudio.com/t/Python-Development-Environments-and-Func/1417720)._

---
I would like them to add more display options from libraries such as Matplotlib and the like, since some commands are not executed as such and sometimes the graphs that one wants to show do not appear.

They could add an interactive window inside the Visual Studio Community in which all the results of the coding of the program appear instead of opening the Python terminal itself.

Visual Studio Community has seemed like a better option than Visual Studio Code to work with Python due to the practicality and power of the IDE to manage PyPi packages and virtual environments, among many other functionalities.

I thank you in advance for your consideration of my suggestion.

---
#### Feedback Bot on 5/10/2021, 11:02 PM: 

Thank you for taking the time to provide your suggestion.  We will do some preliminary checks to make sure we can proceed further.  We'll provide an update once the issue has been triaged by the product team.

**Comment by StellaHuang95** [comment]
Moving this issue to discussion as an enhancement request for comments and upvotes.
